src/xcode/ENA/ENAUITests/ExposureSubmission/ENAUITests_04a_ExposureSubmission.swift

```diff
@@ -440,10 +440,8 @@ class ENAUITests_04a_ExposureSubmission: CWATestCase {
 		app.cells[AccessibilityIdentifiers.ExposureSubmission.TestCertificate.Info.birthdayPlaceholder].waitAndTap()
 
 		// Pick another year to enable the button
-		let dateComponent = Calendar.current.dateComponents([.year], from: Date())
 		let datePicker = XCUIApplication().datePickers.firstMatch
-		let year = try XCTUnwrap(dateComponent.year)
-		datePicker.pickerWheels[String(year)].adjust(toPickerWheelValue: "1985")
```

Comment:
Perhaps add a check to ensure you are on the screen you expect and afterwards, check for existence of the datePicker.